edb/server/protocol/auth_ext/http.py

```diff
@@ -386,7 +388,9 @@ async def handle_token(self, request: Any, response: Any):
             request.url.query.decode("ascii") if request.url.query else ""
         )
         code = _get_search_param(query, "code")
-        verifier = _get_search_param(query, "verifier")
+        verifier = _get_search_param(query, "verifier") or _get_search_param(
+            query, "code_verifier"
+        )
```

Comment:
Use `fallback_keys` here?

Reply:
oh, thank you. I did a search for challenge, but forgot about verifier. Lemme double check and add a test.